api/service/serviceModel.js

```diff
@@ -0,0 +1,34 @@
+const db = require('../../data/db-config');
+
+const findAll = async () => {
+  return await db('services');
+};
+
+const findBy = (filter) => {
+  return db('services').where(filter);
+};
+
+const findById = async (id) => {
+  return db('services').where({ id }).first().select('*');
+};
+
+const create = async (animal) => {
+  return db('services').insert(animal).returning('*');
+};
+
+const update = (id, animal) => {
+  return db('services').where({ id: id }).first().update(animal).returning('*');
```

Comment:
Again I like how you are returning everything for services in create and update, just like you did for animals.